src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/preview.html.twig

```diff
@@ -0,0 +1,154 @@
+                  {{ orderPreview.shippingDetails.carrierName }}
+                {% else %}
+                  -
+                {% endif %}
+              </p>
+
+              <p class="mb-0">
+                <strong>{{ 'Tracking number'|trans({}, 'Admin.Shipping.Feature') }}:</strong>
+                {% if not orderPreview.isVirtual and orderPreview.shippingDetails.trackingNumber is not empty %}
+                  {{ orderPreview.shippingDetails.trackingNumber }}
+                {% else %}
+                  -
+                {% endif %}
+              </p>
+
+              <p class="mb-2">
+                <strong>{{ 'Shipping details'|trans({}, 'Admin.Orderscustomers.Feature') }}:</strong>
+                {% if orderPreview.virtual %}
+                  -
+                {% endif %}
+              </p>
+
+              {% if not orderPreview.virtual %}
+                <p class="mb-0">{{ orderPreview.shippingDetails.firstName }} {{ orderPreview.shippingDetails.lastName }}</p>
+                <p class="mb-0">{{ orderPreview.shippingDetails.address1 }}</p>
+                <p class="mb-0">{{ orderPreview.shippingDetails.address2 }}</p>
+                <p class="mb-0">{{ orderPreview.shippingDetails.country }}</p>
+                <p>{{ orderPreview.shippingDetails.phone }}</p>
+              {% endif %}
+            </div>
+          </div>
+        </div>
+        <div class="col">
+          <div class="row">
+            <div class="col-1">
+              <p class="mb-0">
+                <i class="material-icons pr-1">receipt</i>
+              </p>
+            </div>
+            <div class="col">
+              <p class="mb-1">
+                <strong>{{ 'Invoice details'|trans({}, 'Admin.Orderscustomers.Feature') }}:</strong>
+              </p>
+
+              <p class="mb-0">{{ orderPreview.invoiceDetails.firstName }} {{ orderPreview.invoiceDetails.lastName }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.company }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.address1 }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.address2 }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.country }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.phone }}</p>
+              <p>{{ orderPreview.invoiceDetails.email }}</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="offset-1 col-7">
+      <div class="table-responsive">
+        <table class="table product table-borderless border-bottom-0">
+          <thead>
+          <tr>
+            {% if orderPreview.taxIncluded %}
+              {% set taxInclusion = 'Tax included'|trans({}, 'Admin.Global') %}
+            {% else %}
+              {% set taxInclusion = 'Tax excluded'|trans({}, 'Admin.Global') %}
+            {% endif %}
+
+            <th>{{ 'Product'|trans({}, 'Admin.Global') }}</th>
```

Comment:
Could we also display the number of products ordered like in the mock-up [here](https://projects.invisionapp.com/d/main?origin=v7#/console/18147703/376656277/preview): _Products (%count%)_?